Foundation/URLSession/URLSessionConfiguration.swift

```diff
@@ -209,4 +209,19 @@ open class URLSessionConfiguration : NSObject, NSCopying {
      */
      open var protocolClasses: [AnyClass]?
 
+     /* A Boolean value that indicates whether the session should wait for connectivity to become available, or fail immediately */
+     open var waitsForConnectivity: Bool = false
+
+     /* A service type that specifies the Multipath TCP connection policy for transmitting data over Wi-Fi and cellular interfaces*/
+     open var multipathServiceType: URLSessionConfiguration.MultipathServiceType = .none
+
+}
+
```

Comment:
This should have a warning:

`@available(*,deprecated,message:"Not available on non-Darwin platforms")`

Reply:
We can also use `unavailable` to make it an error to attempt to use it.